<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelayReturnBicycleClass</class>
 <widget class="QDialog" name="DelayReturnBicycleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../bicycle.qrc">:/image/function_background.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 112, 0);</string>
   </property>
   <property name="text">
    <string>延迟还车服务</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DelayReturnBicycleStart">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>580</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>执行延迟还车</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Quit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>580</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DelayReturnBicycleShow">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>931</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>提示：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>660</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../bicycle.qrc">:/image/quit_key_return.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>821</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/DelayReturnBicycleInfo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_rentFlag">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>普通已租车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ticketType">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>普通卡</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_curDealDate">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>1970-01-01 00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_freeTime">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>0分钟</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_emptySae">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>检查中</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_whetherRetrunDelay">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>270</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>否</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_DelayReturnBicycleStart</tabstop>
  <tabstop>pushButton_Quit</tabstop>
 </tabstops>
 <resources>
  <include location="../bicycle.qrc"/>
 </resources>
 <connections/>
</ui>
